smalltalk.html.markdown

```diff
@@ -30,6 +30,7 @@ The most basic operation is to send a message to an object
 `anObject aMessage`
 
 There are three sorts of messages
+
```

Comment:
a single newline char fixes the issues you have raised in #3832 ?

Reply:
Hey Max!

I've checked it over and it does fix the issue raised in #3829